### PR TITLE
Adding head css

### PR DESCRIPTION
**Pull request by ironsidevsquincy**

### CODE REVIEW

File: server/routes/front-page.js [hunk 1]
```diff
@@ -38,16 +41,19 @@ export default region => (req, res) => {
 
 	const sections = getPage('front-page', data, res.locals.flags);
 
+	// get head css
+	const headCss = fs.readFileSync(path.resolve(__dirname, '..', '..', 'public', 'head.css'), 'utf-8');
```
Comment (matthew-andrews) [inline server/routes/front-page.js]: naughty.  this blocks the thread.

Comment (ironsidevsquincy) [inline server/routes/front-page.js]: busted.

I can put some sort of 'read file promise' malarky in n-express, and do it concurrently with the `flagPromise` and `handlebarsPromise`. sound good?

Comment (matthew-andrews) [inline server/routes/front-page.js]: sounds lovely

Comment (matthew-andrews) [inline server/routes/front-page.js]: i dont suppose there's any way to do it without keeping the head css permanently in memory is there?

Comment (ironsidevsquincy) [inline server/routes/front-page.js]: Hmmmm, not sure. Is 100 odd kb an issue?

Comment (matthew-andrews) [inline server/routes/front-page.js]: not for this app

**Pull request merged**